doc/source/conduce-entities.rst

```diff
@@ -0,0 +1,159 @@
+.. _conduce-entities:
+
+================
+Conduce Entities
+================
+
+.. note:: WIP
+
+Conduce uses a concept called entity to define objects that exist in space and time.  An entity can exist for an instant or across a time span.  It can describe a point in space, a path (sequence of points), or a region (polygon).  An entity is defined by a series of samples that describe it's location in space over a period of time.  An entity also has attributes that describe it's characteristics. An entity's attributes can change over time.
+
+-----------------
+Creating entities
+-----------------
+
+An entity is a series of samples that describe the state of and object over a period of time.  A sample is the representation of an entity's momentary state.  Samples are created from user data.  The data should describe people or things that exist in some location during some time span.  Data can be ingested from static files with thousands of records or real-time streams that provide state updates in real-time.  As an example we'll ingest the following real-time shipment data into an existing dataset named ``shipments``:
```

Comment:
2 suggestion for this paragraph:

1) Somewhere in here I'd call out how ID points to an entity, and a sample is info for a specific time.  For example, change the 4th sentence to: "Sample data should identify a person or thing - an entity - and describe its location and state at a given moment in time."

2) When I read this I think, "how do I create an entity?".  To me this sounds like a user has to create an entity & then add samples.  It should be clear that only samples need to be ingested.

Reply:
Yes, the overall language may need to change.  This document is really about samples now.  When we started it was about entities.  In the first sentence we state that, "an entity is a series of samples".

Reply:
I am trying to relate user data to entities/samples here.  That is why I say "the data should describe..." and don't mention entities.  I also don't think IDs are relevant at this more conceptual level.

We are implicitly declaring that entities are unique when we say things like "the state of an object over a period".